web/package.json

```
@@ -84,7 +84,7 @@
     "@storybook/node-logger": "^6.1.11",
     "@storybook/preset-create-react-app": "^3.1.5",
     "@types/jest": "^24.9.1",
-    "cypress": "^5.2.0",
+    "cypress": "6.6.0",
```

Is there a reason we want to lock the minor?
```suggestion
    "cypress": "^6.6.0",
```

---

Good question. I copied the specific update command from somewhere on the cypress site but don't remember where. I'll update to not lock the minor.